### PR TITLE
Expand the FAIR Training Topic, NEW LESSON

### PR DESCRIPTION
**Pull request by kkamieniecka**
Making clinical datasets FAIR: discusses how to make clinical datasets FAIR and why this is important 
@hexylena can we please add logo of UKRI and Elixir UK in trainings and also at the main FAIR training section

### CODE REVIEW

**Comment by hexylena** [comment]
> can we please add logo of UKRI and Elixir UK in trainings and also at the main FAIR training section

yes absolutely! Please just add those images, and we will take the next steps

File: topics/fair/tutorials/fair-clinical/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+>
+{: .agenda}
+
+# Clinical datasets
+
+> <hands-on-title>How would you describe what are clinical datasets? - {% icon time %} 2 minutes - Silent reflection</hands-on-title>
+>
+> * consider in general terms
+> * consider more specifically
+
```
Comment (hexylena) [inline topics/fair/tutorials/fair-clinical/tutorial.md:60]:
```suggestion
```

File: topics/fair/tutorials/fair-clinical/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+> 6. Clear the filters, and search for datasets related to your areas of interest  
+>
+{: .hands_on}
+
+# The importance of making clinical datasets FAIR
+
+## Associations between clinical features and omics
+In some recent work in rheumatoid arthritis [The RA-MAP Consortium, 2022](https://www.nature.com/articles/s41597-022-01264-y), it was demonstrated that simple patient demographics such as sex, ethnicity and age are all drivers of expression variation in addition to disease activity.  Principal component driver plots highlighted critical associations between diverse clinical features and omics.  This shows how rich clinical information may be key to analysis in some, if not, many diseases.  
+
+![RA-MAP.](../../images/RAMAP_graphic.png)
```
Comment (hexylena) [inline topics/fair/tutorials/fair-clinical/tutorial.md:138]: It looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details. *This comment may apply to more images, please consider checking all of them.*

File: topics/fair/tutorials/fair-clinical/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+> For your domain, look at datasets on ArrayExpress and GEO and see how many of them have sex, age, ethnicity/race, and some measure of disease activity
+>
+{: .hands_on}
+
+## Powering up cohorts for analyses and machine learning
+For many diseases (or across a family of diseases), there is a need for well-powered cohorts with hundreds to thousands of participants, for analysis purposes.  FAIR principles have been widely promoted for omics datasets, but as shown, there are key challenges to meta-analysis.  This is illustrated by an example of collating 17 public rheumatoid arthritis studies.  The clinical data in most cases were sparse and incomplete.  For example, of the curated 17 studies, three did not include sex, 5 did not include age and only one study included ethnicity or race.  The limitations in the availability of clinical data substantially diminish the value of public data sets and on the surface to not adhere to FAIR principles.
+
+[comment]: <> (Is there a link to the rheumatoid arthritis study example?)
+[comment]: <> (need to get link from Mike)
+
```
Comment (hexylena) [inline topics/fair/tutorials/fair-clinical/tutorial.md:151]: nice! hadn't seen anyone track their TODOs like this before but it's a nice idea :)

**Comment by kkamieniecka** [comment]
@hexylena, I think all the comments are addressed now. Please let me know is this ok to merge :)

**Comment by hexylena** [comment]
@kkamieniecka since I guess something bad happened on the DaSH end, do you perhaps have a copy of this branch locally and could open a new PR? It seems to have affected them all

**Pull request closed without merging**